scripts/extensions/sams/src/components/assets/assetListPanel.tsx

```diff
@@ -20,6 +20,7 @@ interface IProps {
     onItemClicked(asset: IAssetItem): void;
     onItemDoubleClicked?(asset: IAssetItem): void;
     updateSelectedAssetIds(asset: IAssetItem): void;
+    forceUnlockAsset?(asset: IAssetItem): void;
```

Comment:
Is there a scenario when `forceUnlockAsset` is not provided to `AssetListPanel` on render? (as the `?` tells the compiler this value is optional).

Otherwise the `?` can be removed from `IProps` and `this.props.forceUnlockAsset!(asset)` can be changed to `this.props.forceUnlockAsset(asset)`

Reply:
Yes it is being used in selectAssetModal.

Reply:
Now that this action is controlled by `SamsWorkspaceComponent` this can be removed, along with the class level function `onForceUnlock`

Reply:
Removed :v:  .